controllers/articles-controllers.js

```diff
@@ -9,4 +9,12 @@ const getArticleById = (request, response, next) => {
     })
 }
 
-module.exports = getArticleById
\ No newline at end of file
+const getArticle = (request, response, next) => {
+    fetchArticle().then((articles) => {
```

Comment:
Slightly picky but the function name suggests there's only 1 article. fetchArticles sounds better